supply-chain/audits.toml

```diff
@@ -233,6 +250,23 @@ criteria = "safe-to-deploy"
 version = "0.18.0"
 notes = "I am the author of this crate."
 
+[[audits.gimli]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+version = "0.26.1"
+notes = "This version is used in rust's libstd, so effectively we're already trusting it"
+
+[[audits.gimli]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+delta = "0.26.1 -> 0.27.0"
+
+[[audits.hashbrown]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+delta = "0.12.3 -> 0.13.1"
+notes = "The rust-lang org is the author of this crate"
```

Comment:
Note that despite this it's still good to glance at the changes because I suspect this isn't maintained with a ton of process but rather works the same way that most other open source projects work.

Looking at the diff myself, though, nothing looks amiss. (but would be goo to mention here as well)